bond/bond.py

```diff
@@ -102,6 +102,23 @@ def datalad_undo_last_commit(self):
         reset_proc.check_returncode()
 
     def apply_csv_changes(self, og_prefix, new_prefix):
+        """Applies changes documented in the edited _summary csv
+        and generates the new csv files..
+
+        This function looks at the RenameKeyGroup and MergeInto
+        columns and modifies the bids datset according to the
+        specified changs.
+
+        Parameters:
+        -----------
+            og_prefix : str
+                Path prefix to the directory that contains the
+                original _summary and _files csv files
+            new_prefix : str
```

Comment:
Is this a path to a directory or a file stem? We confused ourselves with this a lot